.travis.yml

```diff
@@ -34,3 +35,8 @@ script:
         source ./.CI/build_all;
       fi
 
+cache:
+    directories:
+        - $HOME/cache
+before_cache:
+    - find $HOME/cache/pkgs/ -mindepth 1 -maxdepth 1 -type d -exec rm -r {} \;
```

Comment:
Thanks, I forgot to handle that. But as I asked on the other PR, wouldn't be better to remove the tarballs instead? This way conda does not need to uncompress them every build